src/test/java/core/basesyntax/service/CsvFileReaderTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CsvFileReaderTest {
+    private static final String PATH_FILE_INTO = "src/main/resources/storage.csv";
+    private CsvFileReader fileReader;
+
+    @Before
+    public void setFileReader() {
+        fileReader = new CsvFileReader();
+    }
+
+    @Test
+    public void fileReader_readSuccess_ok() {
+        List<String> expected = new ArrayList<>();
+        expected.add("b,banana,20");
+        expected.add("b,apple,100");
+        expected.add("s,banana,100");
+        expected.add("p,banana,13");
+        expected.add("r,apple,10");
+        expected.add("p,apple,20");
+        expected.add("p,banana,5");
+        expected.add("s,banana,50");
+        List<String> actual = fileReader.readFromFile(PATH_FILE_INTO);
+        assertArrayEquals(expected.toArray(), actual.toArray());
```

Comment:
You can compare two Lists with assertEquals method. It do the same and you don't have to cast ArrayList into Array